### ISSUE TITLE
Missing publisher information?!?

### ISSUE THREAD

**Issue by musictrivianut** [mylar3/mylar3#1320]
**Describe the bug**
Some series are perpetually in a state of "Comic information is currently being loaded"

**CarePackage**
CarePackage file was *way* too large to attached (500+ MB)

Debug log attached.
[mylar.log](https://github.com/mylar3/mylar3/files/10472781/mylar.log)

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04.1 LTS
 - Branch / commit: master [v0.6.8]

**Additional information**
Continuously throwing WARNINGS that the series do not have a publisher when the information is clearly visible on the ComicVine page.  Guessing that it has to do with the series numbering being odd (one has 05a and 05b, really not sure about the others), but that is seriously just an uneducated guess.

More annoyance than anything, just thought I would point it out, in case someone else has seen it.

**Comment by evilhero** [comment]
You're running 0.6.8, which is very old - update to 0.7 and see if that fixes anything as a first pass. 

Your carepackage should also not be that large, so either your db is over inflated for some reason or you're logging filesize is set to some really large amount.

If it's being loaded continually and only certain series won't refresh - then it's a problem usually related to CV and how the information is therein. Mylar can handle alot of the various numbering conventions, but sometimes stupid is just plain stupid and it can't be automated until the issue arises. If it's every series that won't refresh, then I'd double check you CV API key is correct within Mylar and check permissions on your main comics folder. 

Also, as an aside, just because you can see the info on the CV webpage does not mean that it's accessible the same way in the CV API. If they put publisher in the description instead of in the actual publisher field, then mylar won't pick that up. It will also try to find the imprint if it's an available option given the publisher and year published. So the warning might be just a warning and not an actual error perse.

**Comment by musictrivianut** [comment]
Hmm, okay.  At this point, I am wondering if *I* have done something wrong (probably using git wrong; I have no idea what I'm doing with it).  Clicking "Check for new version" in the GUI says that Mylar is up to date, so I will see if I can figure that out and update it (other than this small glitch, it runs very well).

I cleared the log directory of anything over 30 days and redid the CarePackage thing and it is still 500+ MB (although slightly smaller).  I do have a *very* extensive collection, so that may be why.

I did check the CV page for one of the series and it *looked* like the Publisher information was correct and in the right field, so I'm not sure what is going on.

Thanks for the quick response.  Appreciate it.

**Comment by evilhero** [comment]
Yeah 0.6.8  is not up to date unless you are using Hotio docker, which has been defunct since that version I think and is not to be used. You should be using the lsio docker version if you're using a docker version of mylar.

We have users with hundreds of thousands of comics catalogued within the program and the db is maybe around 100mb tops. Sounds like you may have some extra stuff in the dB that's ballooning it (or it's going corrupt). If you used the import function in mylar to import your existing collection you may have to go back into the import results page and clear the import cache. That may clean things up and lower some space (outside of issuing a 'vaccuum' command in an sqlite command prompt)

Can I ask what the series is that's incorrect?

**Comment by musictrivianut** [comment]
Not running mylar in Docker.  Did copy the git repository files over the old directory files and restarted the service, but version is still showing as 0.6.8.  Really weird.

Checked the Ultron import results and there were only two series listed.  Cleared those.

Stats-wise, this is what I have:

# of Series you're watching: 12747
# of Series you're watching that are continuing: 301
# of Issues you're watching: 106288
# of Issues you actually have: 98333
... total HD-space being used: 3.34 TB

And I have zero knowledge of sqlite, so that is probably out.  Any other suggestions for trying to figure out why the db is so bloated?  Or how to reset it?

Thanks for the help!

**Comment by musictrivianut** [comment]
<sigh>  Sorry, you also asked about the series.

Whiz Comics
The Human Torch
ElfQuest Reader's Collection

Those are the ones that I am seeing the warning on.

**Comment by evilhero** [comment]
Try backing up the mylar.db, config.ini and entire cache directory to a location outside of the Mylar installation. Then delete the entire mylar installation directory.

To do a git install, type in ``git clone https://github.com/mylar3/mylar3`` into the command prompt when you're in the directory where you want it installed.

Once it's installed, copy the files and cache directory you backed up into the new location overwriting anything there. The config.ini and mylar.db should both be in the same directory as Mylar.py.

Open up the config.ini and change the ``git_branch = master``, and set ``log_level = 2`` (to enable debug logging by default).

Start it up, and you should be on v0.7.0.

The rest should all be working at this point - I believe that by copying the git files back on top of an older install there were files that were removed and were still being used.

You may also have to rerun the requirements.txt file to make sure you're current on your python modules.

You can open sqlite from the cmd line, type in ``sqlite3 mylar.db`` when you're in the directory that has the mylar.db file in it.  Once you're in the sqlite console, just type in ``vaccuum`` and hit enter. Wait a few seconds /minutes depending on the speed of your computer, when it's type in ``.exit`` or ``exit`` to quit the sql console. The mylar db should be smaller.

**Comment by musictrivianut** [comment]
If I should open a new issue, please just let me know.  However....

I finally decided to just nuke the .db file and start over, since you said it might be corrupted.  That's fine, I can deal with adding everything again.  Not a big deal.

But, after installing mylar3 per instructions, I'm seeing [None] under version in the settings (screenshot attached;  directories cropped out, but they all point where they should; I checked).

So, what did I do wrong?!?

![mylar version](https://user-images.githubusercontent.com/18358514/217261685-99c730a7-3321-48f9-84e3-99154474fd3c.png)
